feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/widget/SessionsItemDecoration.kt

```diff
@@ -15,14 +15,15 @@ import io.github.droidkaigi.confsched2020.util.AndroidRAttr
 
 class SessionsItemDecoration(
     private val adapter: GroupAdapter<*>,
-    private val context: Context
+    private val context: Context,
+    private val isShowDate: Boolean
```

Comment:
`isShowDate` looks ambiguous because it's shadowed due to [here](https://github.com/DroidKaigi/conference-app-2020/pull/574/files#diff-07d7092decb1657324b9691c9a54e57cR89) and `isShowDateText` is also defined.

How about `visibleSessionDate` for here?